libs/cardano-ledger-binary/src/Cardano/Ledger/Binary/Decoding/Coders.hs

```diff
@@ -425,7 +425,7 @@ hsize (SumD _) = 0
 hsize (RecD _) = 0
 hsize (KeyedD _) = 0
 hsize From = 1
-hsize FromGroup = fromIntegral $ listLenBound $ Proxy @t
+hsize FromGroup = 1
```

Comment:
I am pretty sure this is not right, but not sure how to to go at it

Reply:
I have no knowledge of this field, but it looks to be set to whatever `listLen` is - at least in our code (and found no other references in our github organization besids cardano-ledger). Might we then use `listLen`  here then, I wonder? It would preserve the current behavior , but is it semantically sound?  Wouldn't it have been the choice to begin with, then?  Not sure

Reply:
Oh right, sorry, I didn't look carefully at the arguments of hsize.

Reply:
thanks for looking into this! I generally try to avoid throwing "impossible happened", so I will try with the second more proper solution, even if more involved

Reply:
I'm worried that you had any doubts now 😂